src/App.jsx

```diff
@@ -1,107 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getCommentsForPost(postId) {
+  return commentsFromServer.filter(comment => comment.postId === postId)
+  || null;
```

Comment:
here can be null, filter returns or array with values or empty array
[] || null always be []